README.md

```diff
@@ -1,4 +1,4 @@
-# GitHub Fundamentals for non-developers
+# GitHub Fundamentals
```

Comment:
```suggestion
# GitHub Fundamentals for new players in the sandbox
```

Reply:
I think that mentioning the users will clarify what the reader can expect to find.